orderable/tests/test_models.py

```diff
@@ -207,3 +213,13 @@ def test_changing_parent(self):
         subtask_2.task = task
         subtask_2.save()
         self.assertSequenceEqual(task.subtask_set.all(), [subtask, subtask_2])
+
+    @given(lists(integers(min_value=1), min_size=1, unique=True))
```

Comment:
"I strongly recommend not using `TransactionTestCase` unless you really have to. Because Hypothesis runs this in a loop the performance problems it normally has are significantly exacerbated and your tests will be really slow."  This impacts a majority of our tests, including anything based on a `RequestTestCase`, for instance.